ui/job-view/details/summary/SummaryPanel.jsx

```diff
@@ -18,18 +18,19 @@ export default class SummaryPanel extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!nextProps.selectedJob || !Object.keys(nextProps.selectedJob).length) {
+    const { selectedJob } = nextProps;
+    if (!selectedJob || !Object.keys(selectedJob).length) {
```

Comment:
@camd -- do you know why we need the `.length` check too and not just the `null` check?

Reply:
I think that's a bug.  The null check is sufficient.  I think at some point there must have been a case where in ``defaultProps``, ``selectedJob`` was set to {}.  But that part of the check can be removed.